fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -6341,41 +6349,19 @@ public Map<String, Object> undoLastDisbursal(ScheduleGeneratorDTO scheduleGenera
         return actualChanges;
     }
 
-    /**
-     * Reverse only disbursement, accruals, and repayments at disbursal transactions
-     *
-     * @param actualDisbursementDate
-     * @return
-     */
-    public List<LoanTransaction> reverseExistingTransactionsTillLastDisbursal(LocalDate actualDisbursementDate) {
```

Comment:
Accrual and repayment at disbursal can be reversed even if it was after the last disbursement.
Please be more careful to not do any regression!

Reply:
Done